.github/workflows/release.yml

```diff
@@ -122,14 +122,14 @@ jobs:
       - name: Build WASI-NN plugin
         shell: bash
         run: |
-          source /opt/intel/openvino_2021/bin/setupvars.sh
           export Torch_DIR=$(pwd)/${PYTORCH_INSTALL_TO}/libtorch
           ldconfig
           plugin_array=(${tar_names})
           option_array=(${build_options})
           outtarget=${output_bin%.*}
           outtarget=${outtarget#lib}
-          cmake -Bbuild -GNinja -DCMAKE_BUILD_TYPE=Release -DWASMEDGE_BUILD_AOT_RUNTIME=OFF -DWASMEDGE_BUILD_TOOLS=OFF
+          cmake -Bbuild -GNinja -DCMAKE_BUILD_TYPE=Release -DWASMEDGE_BUILD_AOT_RUNTIME=OFF -DWASMEDGE_BUILD_TOOLS=OFF 
+
```

Comment:
It looks like these two lines do nothing. Why do we need to change this?

Reply:
These lines were originally present. Should I remove them?

Reply:
No. You don't need to modify them, but you did.

Reply:
Sorry . I modified them accidentally while testing but I have restored the changes

Reply:
Please recover it to avoid the modifications. You can check if this is recovered by the PR page. The modified part will be marked as green lines.

Reply:
I removed the unnecessary modification